src/main/java/net/apnic/rdap/resource/StaticResourceLocator.java

```diff
@@ -0,0 +1,62 @@
+package net.apnic.rdap.resource;
+
+import net.apnic.rdap.authority.RDAPAuthority;
+
+/**
+ * Class provides a static implementation of the ResourceLocator interface.
+ *
+ * This class will either always return the same RDAPAuthority or always
+ * throw a ResourceNotFoundException.
+ *
+ * @param <Resource> The type of resource this class assists with
+ */
+public class StaticResourceLocator<Resource>
```

Comment:
Purpose of this class is to eventually provide a configuration means by which we can return an authority for things that are harder to route/proxy for.

For example at the moment there is no way to know which authority to route entity requests to. Its my idea that in future stories there will be a config property that says for all entity requests send them to this authority. In the configuration stage of the application that authority would get plugged into this class for the entity locator.